ws-nextjs-app/pages/api/local/[...id].api.ts

```diff
@@ -0,0 +1,54 @@
+import fs from 'node:fs/promises';
+import path from 'path';
+import type { NextApiRequest, NextApiResponse } from 'next';
+
+const getFixtureLocation = (pageType: string) =>
+  ({
+    live: 'livePage',
```

Comment:
Is there any chance that this could be renamed (and the fixtures moved) to `live` instead of `livePage`? Just for consistency / better dev-x because we can find things more easily..?

Reply:
I think this is probably what Aaron is also discussing https://github.com/bbc/simorgh/pull/11753/files#r1700322809

Reply:
Thanks for the feedback Karina.
Yes. This is feasible. I adopted the `livePage` directory name based
on what's currently on [latest](https://github.com/bbc/simorgh/tree/latest/data/pidgin/livePage) 
We can adopt the name, `live`, directory though and remove the need to infer it using
[getFixtureLocation](https://github.com/bbc/simorgh/pull/11753/files#diff-f00bd0ff3cac1838b7acb0280961815efa369af041f44f65e551d6b2196e9364R5).
I'll proceed to make this change.